test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -428,11 +432,14 @@ public async Task OutputsValidJSONLogs()
 
                 // (2) Contains minimal expected fields
                 List<string> keys = json.Properties().Select(p => p.Name).ToList();
+                Assert.Contains("EventStampName", keys);
+                Assert.Contains("EventPrimaryStampName", keys);
+                Assert.Contains("ProviderName", keys);
+                Assert.Contains("TaskName", keys);

```

Comment:
the rest of this PR should enables us to log these 4 properties in linux. That's it!